control-plane/catalog/to-consul/syncer.go

```diff
@@ -433,14 +456,28 @@ func (s *ConsulSyncer) syncFull(ctx context.Context) {
 			"node-name", r.Node,
 			"service-id", r.ServiceID,
 			"service-consul-namespace", r.Namespace)
+
 		_, err = consulClient.Catalog().Deregister(r, nil)
 		if err != nil {
+			// metric count for error syncing K8S services with Consul via calatog sync
+			labels := []metrics.Label{
+				{Name: "error", Value: err.Error()},
+			}
+			s.PrometheusSink.IncrCounterWithLabels([]string{"consul", "sync_catalog", "to_consul", "deregister", "error"}, 1, labels)
```

Comment:
can we assign these slices to variables and reuse them throughout the code so if we change anything it's just changing it in one place?